BUILD.md

```diff
@@ -253,6 +253,7 @@ on/off options currently supported by this repository:
 | LOADER_ENABLE_ADDRESS_SANITIZER          | Linux & macOS | `OFF`   | Enables Address Sanitizer in the loader and tests.                                                                                                                                |
 | LOADER_ENABLE_THREAD_SANITIZER           | Linux & macOS | `OFF`   | Enables Thread Sanitizer in the loader and tests.                                                                                                                                 |
 | LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING | All           | `OFF`   | Causes the loader to not unload dynamic libraries. Example use case. This option allows leak sanitizers to have full stack traces.       |
+| LOADER_USE_UNSAFE_FILE_SEARCH            | All           | `OFF`   | Allows the loader to search in unsafe locations. |
```

Comment:
```suggestion
| LOADER_USE_UNSAFE_FILE_SEARCH            | All           | `OFF`   | Disables security policies that prevent unsecure locations from being used when running with elevated permissions.  |
```

Reply:
I kinda want there to be an additional note saying that this should not be enabled except in very specific contexts, like isolated test environments. (but alas, that might be a 'well duh!' kinda comment so is a bit redundant)

Reply:
Sure. I removed the ccache section to make room for the explicit security note.

Reply:
My intent being to try and trim down the documentation if I add new documentation.